src/Sentry.Protocol/User.cs

```diff
@@ -45,6 +47,15 @@ public class User
         [DataMember(Name = "username", EmitDefaultValue = false)]
         public string Username { get; set; }
 
+        /// <summary>
+        /// Additional informations about the user
+        /// </summary>
+        /// <value>
+        /// Additional informations about the user
+        /// </value>
+        [DataMember(Name = "extras", EmitDefaultValue = false)]
```

Comment:
Please serialize this as `other` to match the other SDKs.
Property name should be `Other`.